osu.Game.Rulesets.Karaoke/KaraokeRuleset.cs

```diff
@@ -284,12 +284,14 @@ public override StatisticItem[] CreateStatisticsForScore(ScoreInfo score, IBeatm
         return statistic.ToArray();
     }
 
-    public override IEnumerable<SetupSection> CreateEditorSetupSections() => new SetupSection[]
-    {
+    public override IEnumerable<Drawable> CreateEditorSetupSections() =>
+    [
+        new MetadataSection(),
+        new ResourcesSection(),
         new KaraokeSingerSection(),
         new KaraokeTranslationSection(),
         new KaraokeNoteSection(),
-    };
+    ];
```

Comment:
Note:
Still not decide using the latest net language for the array, so please change back to []{ ... }